pkg/clusteragent/admission/mutate/config/config.go

```diff
@@ -177,6 +142,42 @@ func (w *Webhook) WebhookFunc() admission.WebhookFunc {
 // - DD_EXTERNAL_ENV: the External Data Environment Variable
 func (w *Webhook) inject(pod *corev1.Pod, _ string, _ dynamic.Interface) (bool, error) {
 	var injectedConfig, injectedEntity, injectedExternalEnv bool
+	var (
```

Comment:
Any reason to move this section?

Reply:
These variables rely on the datadog configuration to be created. Originally they were created in the global file scope using the pkgconfigsetup, however, we are intending on removing pkgconfigsetup so I moved the instantiation of the variables into the relevant function they are being used and where the datadog config component exists.